tests/testthat/test-target_market_share.R

```diff
@@ -676,3 +676,52 @@ test_that("for one company with multiple loans of different size, unweighted
 
   expect_equal(projected$production, fake_ald()$production)
 })
+
+test_that("`technology_share` outputs consistently when multiple
+          direct_loantakers match to a single company (#265)", {
+
+  matched <- fake_matched(
+    id_loan = c("L1", "L2", "L3", "L4", "L5"),
+    name_ald = c(rep("company a", 4), "company b")
+  )
+
+  matched_split_dl <- matched %>%
+    mutate(name_direct_loantaker = c("company a1", "company a2", "company a3", "company a4", "company b"))
```

Comment:
Is this 80 characters or less?

Reply:
I suggest:

``` r
c(paste0("company a", 1:4), "company b")
#> [1] "company a1" "company a2" "company a3" "company a4" "company b"
```

<sup>Created on 2021-01-21 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>